lib/client-assets.php

```diff
@@ -686,6 +686,7 @@ function gutenberg_editor_scripts_and_styles( $hook ) {
 		sprintf( '/wp/v2/users/me?post_type=%s&context=edit', $post_type ),
 		array( '/wp/v2/media', 'OPTIONS' ),
 		array( '/wp/v2/blocks', 'OPTIONS' ),
+		sprintf( '/wp/v2/%s/%s/autosaves?context=edit', $rest_base, $post->ID ),
```

Comment:
I dunno how much we care to use typed placeholders, but I see no reason why the second ought not be `%d`. The same could be said for the `sprintf( '/wp/v2/%s/%s?context=edit', $rest_base, $post->ID )` a few lines up, too.

Reply:
FYI: All this code will be removed as of #13569 . You should be prepared to submit an upstream patch via Trac. Do we need it to exist in the meantime in Gutenberg? If so, we should start thinking about how we can extend this from the plugin (i.e. the `block_editor_preload_paths` filter).